website/views/company.py

```
@@ -576,12 +609,150 @@
+
+        # Redirect to the actual social media URL
+        return redirect(target_url)
+
+
+class OrganizationProfileEditView(View):
+    """
+    View for editing organization profile information and social media links.
+    """
+
+    @validate_organization_user
+    def get(self, request, id, *args, **kwargs):
```

Ruff (0.14.5)

Unused method argument: `args`

(ARG002)